src/classes/UTIL_OrgTelemetry_SObject_BATCH.cls

```diff
@@ -113,14 +136,33 @@ public class UTIL_OrgTelemetry_SObject_BATCH implements Database.Batchable<SObje
         }
     }
 
+    /**
+     * @description Process RecurringDonation counters for the specific Opportunity
+     */
+    private void getRecurringDonationsCounts(npe03__Recurring_Donation__c recurringDonation) {
+        rdAllCount++;
+
+        for (Opportunity opp : recurringDonation.npe03__Donations__r) {
+            if (opp.Amount != recurringDonation.npe03__Amount__c) {
+                rdDiffCount++;
```

Comment:
The quip dock mentioned using the original job to count the recurring donation diffs, but I thought it would be better to do this in the new job so we don't have to keep track of the unique recurring donations.

Reply:
Agreed. Good idea.